ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/Runner.java

```diff
@@ -95,9 +98,11 @@ public Runner(
     this.vertx = vertx;
     this.allowedCorsOrigins = allowedCorsOrigins;
     this.serverOptions = serverOptions;
+    this.metricsEndpoint = metricsEndpoint;
   }
 
   public void start() throws ExecutionException, InterruptedException {
+    metricsEndpoint.start(vertx);
```

Comment:
do we need to publish metrics port to the ports file?

Reply:
done